src/coreclr/jit/importervectorization.cpp

```diff
@@ -550,6 +551,12 @@ GenTree* Compiler::impStringEqualsOrStartsWith(bool startsWith, CORINFO_SIG_INFO
     const bool isStatic  = methodFlags & CORINFO_FLG_STATIC;
     const int  argsCount = sig->numArgs + (isStatic ? 0 : 1);
 
+    // This optimization spawns several temps so make sure we have a room
+    if (lvaHaveManyLocals(0.75))
+    {
+        JITDUMP("impSpanEqualsOrStartsWith: Method has too many locals - bail out.")
+    }
```

Comment:
Missing return?

Reply:
nit: do you need a trailing `\n`?

Reply:
Addressed both issues, thanks! That explains why I didn't see diffs from that 😆